pattern-2/bosh-release/config/blobs.yml

```diff
@@ -1,20 +1,28 @@
+jdbcdrivers/mssql-jdbc-7.0.0.jre8.jar:
```

Comment:
Shall we commit an empty file when sending a PR? IMO, updating this with every PR does not make sense.